README.md

```diff
@@ -1,189 +1,64 @@
-| [Úll](http://2017.ull.ie/) | **April 10-11, 2017** | 🇮🇪 Killarney, Ireland|
-| [Mobius Conference](http://mobiusconf.com/en/) | **April 21-22, 2017** | 🇷🇺 Saint-Petersburg, Russia |
-| [App Builders](http://www.appbuilders.ch) | **April 24-25, 2017** | 🇨🇭 Lausanne, Switzerland |
-| [Craft Conference](http://craft-conf.com) | **April 25-28, 2017** | 🇭🇺 Budapest, Hungary |
-| [MCE](http://2017.mceconf.com/) | **May 8-9, 2017** | 🇵🇱 Warsaw, Poland |
-| [DEVit Web Conference](http://devitconf.org/) | **May 20-21, 2017** | 🇬🇷 Thessaloniki, Greece |
-| [App Design & Development Conference](http://addconf.com/) | **June 22-23, 2017** | 🇪🇸 Barcelona, Spain |
-| [Mobile Era](https://mobileera.rocks/) | **October 5-6, 2017** | 🇳🇴 Oslo, Norway |
-| [Mobiconf](https://2017.mobiconf.org/) | **October 5-6, 2017** | 🇵🇱 Krakow, Poland |
-| [Release Notes](https://2017.releasenotes.tv/) | **October 16-18, 2017** | 🇺🇸 Chicago, IL, USA |
-| [Mobilization](http://2017.mobilization.pl/) | **October 21, 2017** | 🇵🇱 Łódź, Poland |
-| [Mobius](https://mobiusconf.com/en/) | **November 11, 2017** | 🇷🇺 Moscow, Russia |
-| [DevFest Baltics](https://devfest.gdg.lv/) | **November 17, 2017** | 🇱🇻 Riga, Latvia |
-| [UA Mobile](http://www.uamobile.org/) | **November 25, 2017** | 🇺🇦 Kyiv, Ukraine |
-| [MobOS](http://romobos.com/) | **February 15-16, 2018** | 🇷🇴 Cluj-Napoca, Romania |
-| [MobileTech](https://mobiletechcon.de/en/) | **March 12-14, 2018** | 🇩🇪 Munich, Germany | n/a |
-| [Appdevcon](http://appdevcon.nl/) previously called mDevcon | **March 15-18, 2018** | 🇳🇱 Amsterdam, Netherlands |
-| [Code Mobile](http://www.codemobile.co.uk) | **April 2-5, 2018** | 🇬🇧 Chester, UK |
-| [App Builders](https://www.appbuilders.ch/) | **April 16-17, 2018** | 🇨🇭 Lugano, Switzerland |
-| [Craft Conference](http://craft-conf.com) | **May 8-11, 2018** | 🇭🇺 Budapest, Hungary |
-| [MCE](http://2018.mceconf.com) | **June 5-6, 2018** | 🇵🇱 Warsaw, Poland | 
-| [ADDC - App Design & Development Conference](https://addconf.com/) | **July 4-6, 2018** | 🇪🇸 Barcelona, Spain |
-| [Mobiconf](https://2018.mobiconf.org/) | **October 4-5, 2018** | 🇵🇱 Krakow, Poland |
-| [Mobile Era](https://mobileera.rocks/) | **November 1-2, 2018** | 🇳🇴 Oslo, Norway |
-| [DevFest Baltics](https://devfest.gdg.lv/) | **November 16** | 🇱🇻 Jurmala, Latvia |
-| [Functional Swift Conference 2018](http://2018.funswiftconf.com) | **December 1, 2018** | 🇺🇸 New York, NY |
-
-### More
-
-* A [Twitter list](https://twitter.com/NeoNacho/lists/cocoaconferences) with all the conferences made by [Boris Bügling](https://twitter.com/NeoNacho)
-
-If you want to add a conference to this list or edit the info, send a **pull request**, [ping me on twitter](https://twitter.com/lascorbe) or get in touch at [luis@nsspain.com](mailto:luis@nsspain.com). Thank you!
+[http://ascor.be/CocoaConferences/](http://ascor.be/CocoaConferences/)
```

Comment:
I'd put clearly this is the only way to see the conferences list now, something like:
```
## To see the list of conferences go here:
[http://ascor.be/CocoaConferences/](http://ascor.be/CocoaConferences/)
```

Reply:
how's this? https://github.com/davedelong/CocoaConferences/blob/master/README.md

Reply:
Perfect!